build.gradle

```diff
@@ -6,7 +6,8 @@ plugins {
   id 'com.github.ben-manes.versions' version '0.42.0'
 
   id 'com.diffplug.spotless' version '6.4.2' apply false
-  id 'io.freefair.lombok' version '6.4.2' apply false
+  id 'io.freefair.aspectj.post-compile-weaving' version '6.4.3' apply false
```

Comment:
That's really all it takes to enable AspectJ in Gradle, I was positively surprised 😯